lib/http-context.js

```diff
@@ -38,6 +41,8 @@ function HttpContext(req, res, method, options) {
   this.args = this.buildArgs(method);
   this.methodString = method.stringName;
   this.supportedTypes = this.options.supportedTypes || DEFAULT_SUPPORTED_TYPES;
+  var streamDesc = method.stream;
+  var methodReturnsStream = !!streamDesc;
```

Comment:
IIUC, the stream mode is turned on by marking a return argument as `type: 'stream'`. This code seems to check the method metadata instead?